src/main/resources/openapiv2.json

```diff
@@ -671,75 +802,130 @@
-                "value": {
-                    "type": "object"
+                "fetch.min.bytes": {
+                    "description": "Sets the minimum ammount of data in bytes for the consumer to receive. The broker waits until the data to send exceeds this amount.",
+                    "type": "string"
                 },
-                "key": {
-                    "type": "object"
+                "consumer.request.timeout.ms": {
+                    "description": "Sets the maximum amount of time in milliseconds (ms) for the consumer to wait for messages for a request. If the timeout period is reached without a response, an error is returned.",
+                    "type": "string"
                 }
+            },
+            "example": {
+                "name": "consumer1",
+                "format": "binary",
+                "auto.offset.reset": "earliest",
+                "auto.commit.enable": "false",
+                "fetch.min.bytes": "512",
+                "consumer.request.timeout.ms": "30000"
             }
         },
-        "Topics": {
-            "title": "Topics",
+        "Offsets": {
+            "title": "Offsets",
             "type": "object",
             "properties": {
-                "topics": {
+                "offsets": {
                     "type": "array",
                     "items": {
-                        "type": "string"
+                        "$ref": "#/definitions/Offset"
                     }
                 }
             },
-            "example": "{\n    \"topics\": [\n        \"topic1\",\n        \"topic2\"\n    ]\n}"
+            "example": {
+                "offsets": [
+                    {
+                        "topic": "topic",
+                        "partition": 0,
+                        "offset": 15
+                    },
+                    {
+                        "topic": "topic",
+                        "partition": 1,
+                        "offset": 42
+                    }
+                ]
+            }
         },
-        "Records": {
-            "title": "Records",
+        "OffsetsWithoutTopic": {
+            "title": "Offsets",
             "type": "object",
             "properties": {
-                "records": {
+                "offsets": {
                     "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/Record"
+                    "items": {}
+                }
+            },
+            "example": {
+                "offsets": [
+                    {
+                        "partition": 2,
+                        "offset": 0
+                    },
+                    {
+                        "partition": 1,
+                        "offset": 1
+                    },
+                    {
+                        "partition": 2,
+                        "offset": 2
                     }
+                ]
+            }
+        },
+        "Partition": {
```

Comment:
I don't think we use this one, it contains information about administration things like who is the leader etc etc. What am I missing?